docs/user-guide/installation.rst

```diff
@@ -9,42 +9,185 @@ Installation
 
 .. edit-link::
 
-The recommended way to get started with Rasa is via ``pip``:
+Quick Installation
+~~~~~~~~~~~~~~~~~~
 
-.. copyable::
+You can install both Rasa and Rasa X using pip with the following command (requires Python 3.5.4 or higher).
 
-    pip install rasa-x --extra-index-url https://pypi.rasa.com/simple
+.. code-block:: bash
```

Comment:
let's use the `copyable` directive for these instead of code-block

Reply:
This is fine for single line code snippets, but look very bad for multilined ones. That's why I switched to code blocks for consistency.